packages/manager/modules/sms/src/sms/compose/translations/Messages_fr_FR.xml

```diff
@@ -76,7 +76,14 @@
 
    <translation id="sms_sms_compose_status_success" qtlid="341732">Votre SMS a bien été envoyé</translation>
    <translation id="sms_sms_compose_status_failed" qtlid="341745">Une erreur est survenue lors de l’envoi de votre SMS</translation>
-   <translation id="sms_sms_compose_status_invalid_receivers" qtlid="341758">Votre SMS n’a pu être envoyé aux numéros suivants&#160;:&#160;</translation>
+   <translation id="sms_sms_compose_invalid_receivers">destinataires invalides</translation>
+   <translation id="sms_sms_compose_invalid_receiver">destinataire invalide</translation>
+   <translation id="sms_sms_compose_receivers">aux destinataires</translation>
+   <translation id="sms_sms_compose_receiver">au destinataire</translation>
+   <translation id="sms_sms_compose_receiver_numbers">aux numéros suivants</translation>
+   <translation id="sms_sms_compose_receiver_number">au numéro suivant</translation>
+   <translation id="sms_sms_compose_status_invalid_receivers">Votre SMS n’a pu être envoyé {{receiverNumber}} : {{invalidReceivers}} ({{invalidCause}})</translation>
```

Comment:
Assuming there is a large amount of invalid records from a receivers list it will result with a quite long warning message.

Existing any threshold for displaying that kind of information?

In my point of view for large report we should display a more relevant view.

Is there a CX topic around it?

Thanks!

Reply:
I need to notify users if they have sent SMS to multiple recipients and there are errors, which are errors.

Reply:
I agree that we have to display that information to the user but is the case where you are dealing with a large invalid receivers' list has been addressed?

Reply:
ping @Steffy29 Do you have any feedbacks regarding my previous comments?